samples/slack/README.md

```diff
@@ -24,39 +24,29 @@ Slack API documentation:
 This project isn't meant to cover all available functions and events. It
 merely showcases a few illustrative, annotated, reusable examples.
 
-## Instructions
+## How It Works
 
-1. Deploy the manifest file:
+1. Listen for Slack events such as mentions, slash commands, new messages, edited messages, and emoji reactions.
+2. Process the event data to extract relevant information (e.g., user, message, or action details).
+3. Send responses or updates messages in Slack channels or threads based on the event type.
 
-   ```shell
-   ak deploy --manifest samples/slack/autokitteh.yaml
-   ```
+## Cloud Usage
 
-2. Follow the instructions in the `ak` CLI tool's output:
+1. Initialize your connection with Slack
+2. Deploy project
 
-   ```
-   Connection created, but requires initialization.
-   Please run this to complete:
+## Trigger Workflow
 
-   ak connection init <connection ID>
-   ```
+- Mention the Slack app in a message (e.g., "Hi @autokitteh") to trigger a response
+- Interact with a Slack block element (e.g., click a button or select from a dropdown)
+- Send a message or reply in a Slack channel
+- Add a reaction (emoji) to a message in Slack
+- Use a registered slash command (e.g., `/autokitteh <channel name or ID>`)
+> [!IMPORTANT]  
+> Self-hosted users who set up their Slack App using https://docs.autokitteh.com/integrations/slack/connection will have their commands registered as expected. If you set up your app differently, your registered commands may vary.
+> [!NOTE]
```

Comment:
There should be an empty line before and after alert notes in GitHub markdown. In this specific case the lack of an empty line actually corrupts the rendering:

https://github.com/autokitteh/kittehub/blob/bdd3b6f2bb183d9df7705db3acd9141658bdecba/samples/slack/README.md#trigger-workflow

```suggestion

> [!NOTE]
```